code/datums/status_effects/debuffs.dm

```diff
@@ -390,6 +390,20 @@
 			I.take_damage(100)
 	return ..()
 
+/// A status effect used for specifying confusion on a living mob.
+/// Created automatically with /mob/living/set_confusion.
+/datum/status_effect/confusion
+	id = "confusion"
+	alert_type = null
+	var/strength
+
+/datum/status_effect/confusion/tick()
+	strength -= 1
+	if (strength <= 0)
+		owner.remove_status_effect(STATUS_EFFECT_CONFUSION)
+		return
+	strength -= 1
```

Comment:
```suggestion
	if (--strength <= 0)
		owner.remove_status_effect(STATUS_EFFECT_CONFUSION)
		return
```
I assume you don't want to diminish 2 per tick, right?

Reply:
I didn't mean to drain 2 per tick, no, good catch. I'm going to keep `strength -= 1` though since I prefer it to be more readable.